src/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts

```diff
@@ -26,5 +26,8 @@ export function getWorksForSaleRouteVariables({ artistID }, { location }) {
     aggregations,
     artistID,
     includeBlurHash: false,
+    // If the route was prefetched, we want `isPrefetching: true` so the query
+    // can be resolved immediately.
+    isPrefetching: !!location?.query?.is_prefetched,
```

Comment:
#superminor, can we update this and keep the naming 1:1 with JS case? `artist/id?isPrefetching=true

Reply:
So there's a reason I rolled with different variables - let me try to explain. Not sure if it's worth it or not.

The GraphQL variable `isPrefetching` is being considered...when the query is in the process of being prefetched or not. That's happening currently, so `isPrefetching` (the `-ing` form makes sense to me).

But then, updating the link, that is only considered later _after_ prefetching, whenever the user has actually navigated via link. At that point, the signal we're trying to send is 'was this already successfully prefetched' - so i went with the `-ed` form of the word 'prefetch'. `isPrefetching` vs `hasBeenPrefetched` kind of thing.